### DISCUSSION TITLE
Add CLI argument to prevent `medusa develop` from opening a new browser tab

### DISCUSSION THREAD

**Discussion by MorganeLeCaignec** [medusajs/medusa#7309] [Feature Requests]
This is fairly standard in dev/develop/start scripts, an option or argument to prevent the `medusa develop` script from automatically opening a new browser window/tab, which gets annoying pretty quickly when running the script multiple times or when the detected default browser/window isn't the correct one.

**Comment by olivermrbl** [comment]
You can disable this in the [admin plugin configuration](https://docs.medusajs.com/admin/configuration#plugin-options).
```ts
const plugins = [
  // ...
  {
    resolve: "@medusajs/admin",
    options: {
      develop: {
        open: false,
      },
    },
  },
]

```
